### PR TITLE
Ref: "SKIP_HEADER_ISSUE" PR #710

### PR DESCRIPTION
**Pull request by hpandeycodeit**
Fixed a failing integration tests (GPDB 5) for the PR #710. 

Without setting these options: 
```
\x on
\pset format unaligned
``` 

The query in GPDB 6 returns output in this format ( notice the `+` appended in the second column) : 

```
select * from pxf_multi_csv_with_header;
 num1 |  word  | num2
------+--------+------
    1 | aaa_1 +|    2
      | _c     |
    2 | aaa_2  |    3
    3 | aaa_3 +|    4
      | _c     |
    4 | aaa_4  |    5
    5 | aaa_5 +|    6
      | _c     |
```

And the same query returns the output like this in GPDB 5: 

```
select * from pxf_multi_csv_with_header;
 num1 |  word  | num2 
------+--------+------
    1 | aaa_1  |    2
      : _c       
    2 | aaa_2  |    3
    3 | aaa_3  |    4
      : _c       
    4 | aaa_4  |    5
    5 | aaa_5  |    6
      : _c       
```

Once we set extended display on with format unaligned, the output comes in the same format in both GPDB 5 and GPDB 6: 

```
\x on
\pset format unaligned

select * from pxf_multi_csv_with_header;
num1|1
word|aaa_1
_c
num2|2

num1|2
word|aaa_2
num2|3

num1|3
word|aaa_3
_c
num2|4

num1|4
word|aaa_4
num2|5
```

### CODE REVIEW

File: automation/tincrepo/main/pxf/features/hdfs/readable/text/multiline_csv_data_with_header/expected/query01.ans [hunk 1]
```diff
@@ -6,487 +6,1449 @@ SELECT COUNT(*) from pxf_multi_csv_with_header;
  320
 (1 row)
 
+-- Display on for output consistency between GPDB 5 and 6
+\x on
+Expanded display is on.
+\pset format unaligned
+Output format is unaligned.
```
Comment (ashuka24) [inline automation/tincrepo/main/pxf/features/hdfs/readable/text/multiline_csv_data_with_header/expected/query01.ans:13]: why are we doing it unaligned with expanded display now? it is much harder to parse this now.

Comment (hpandeycodeit) [reply automation/tincrepo/main/pxf/features/hdfs/readable/text/multiline_csv_data_with_header/expected/query01.ans:13]: Without this, these tests were failing for GPDB 5

Comment (bradfordb-vmware) [reply automation/tincrepo/main/pxf/features/hdfs/readable/text/multiline_csv_data_with_header/expected/query01.ans:13]: Can we include more detail (either here as a comment or in the commit message) about what the differences are in the output for GPDB5 vs GPDB6?

Comment (hpandeycodeit) [reply automation/tincrepo/main/pxf/features/hdfs/readable/text/multiline_csv_data_with_header/expected/query01.ans:57]: Thanks! Added in the comments

**Review by denalex — APPROVED** [review]

**Review by ashuka24 — APPROVED** [review]

**Pull request merged**